app/src/main/java/view/GithubUsersView.java

```diff
@@ -0,0 +1,9 @@
+package view;
+
+import java.util.List;
+
+import model.GithubUsers;
+
+public interface GithubUsersView {
+    void githubUsersReady(List<GithubUsers> githubUsers);
```

Comment:
Please try to follow the [clean code conventions](https://andela.instructure.com/courses/282/files/22422/download?verifier=JdA53jXl4Q9FgM4zWuevMBB1Dli95Pt0yyOBnu7S&wrap=1) 
when naming methods and variables